lib/settings_base.py

```diff
@@ -80,6 +80,9 @@ def path(*folders):
 DATABASES['default']['TEST_CHARSET'] = 'utf8'
 DATABASES['default']['TEST_COLLATION'] = 'utf8_general_ci'
 
+# Run all views in a transaction unless they are decorated not to.
+ATOMIC_REQUESTS = True
```

Comment:
So if you had:

```py
def change_slug(request, pk):
    addon = Addon.objects.get(pk=pk)
    addon.update(slug='a-new-slug')
    return 200
```

The `addon` would get selected from the replica without a transaction and then it would do the update within a transaction and commit it upon returning?